odl/tomo/data/mrc2014.py

```diff
@@ -20,72 +20,75 @@
-|         |       |         |        |the "unit cell"                |
-+---------+-------+---------+--------+-------------------------------+
-|10       |37-40  |Int32    |MZ      |Number of intervals along Z of |
-|         |       |         |        |the "unit cell"                |
-+---------+-------+---------+--------+-------------------------------+
-|11-13    |41-52  |Float32  |CELLA   |Cell dimension in angstroms    |
-+---------+-------+---------+--------+-------------------------------+
-|...      |       |         |        |                               |
-+---------+-------+---------+--------+-------------------------------+
-|20       |77-80  |Float32  |DMIN    |Minimum density value          |
-+---------+-------+---------+--------+-------------------------------+
-|21       |81-84  |Float32  |DMAX    |Maximum density value          |
-+---------+-------+---------+--------+-------------------------------+
-|22       |85-88  |Float32  |DMEAN   |Mean density value             |
-+---------+-------+---------+--------+-------------------------------+
-|23       |89-92  |Int32    |ISPG    |Space group number 0, 1, or 401|
-+---------+-------+---------+--------+-------------------------------+
-|24       |93-96  |Int32    |NSYMBT  |Number of bytes in extended    |
-|         |       |         |        |header                         |
-+---------+-------+---------+--------+-------------------------------+
-|...      |       |         |        |                               |
-+---------+-------+---------+--------+-------------------------------+
-|27       |105-108|Char     |EXTTYPE |Extended header type           |
-+---------+-------+---------+--------+-------------------------------+
-|28       |109-112|Int32    |NVERSION|Format version identification  |
-|         |       |         |        |number                         |
-+---------+-------+---------+--------+-------------------------------+
-|...      |       |         |        |                               |
-+---------+-------+---------+--------+-------------------------------+
-|50-52    |197-208|Float32  |ORIGIN  |Origin in X, Y, Z used in      |
-|         |       |         |        |transform                      |
-+---------+-------+---------+--------+-------------------------------+
-|53       |209-212|Char     |MAP     |Character string 'MAP' to      |
-|         |       |         |        |identify file type             |
-+---------+-------+---------+--------+-------------------------------+
-|54       |213-216|Char     |MACHST  |Machine stamp                  |
-+---------+-------+---------+--------+-------------------------------+
-|55       |217-220|Float32  |RMS     |RMS deviation of map from mean |
-|         |       |         |        |density                        |
-+---------+-------+---------+--------+-------------------------------+
++---------+--------+----------+--------+-------------------------------+
```

Comment:
Because the table will look much better when used in documentation (once I figure out how to add it). And it's not that much harder to parse it than to process the list.